.ci/Jenkinsfile

```diff
@@ -15,13 +15,14 @@
 // specific language governing permissions and limitations
 // under the License.
 
-@Library('apm@main') _
+@Library('apm@use-junit2otel-step') _
```

Comment:
To be reverted after it gets merged?

Reply:
How do you usually do with kind of use cases, where we need a custom branch to verify the new behaviours?

Reply:
1) PR from my fork (with the steps/changes but without changing the Jenkinsfile)
2) Branch called `test/....` in the upstream with the changes in the Jenkinsfile
3) If the branch works, then I merge my PR
4) Create a branch from 2) with the changes in the Jenkinsfile.

Another approach could be:
1) PR from my fork
2) Create pipeline in the local docker jenkins instance (and use those new steps)
3) Merge the PR
4) Create PR with the changes in the Jenkinsfile

Reply:
Got it. I think this branch is following the first approach, so I can:
1. revert the usage of the new step in the Jenkinsfile
2. merge this PR
3. release the library
4. create a new PR with the changes in the Jenkinsfile

Does it sound good?

Reply:
> release the library

You don't need this, it uses `main` :)

But  I need to release a new version shortly, so I'll wait for you

Reply:
Could doing it asap

Reply:
@v1v I removed any change in the Jenkinsfile regarding the new step. I think this is ready to be merged